contracts/interfaces/draft-IERC2612.sol

```diff
@@ -1,8 +1,8 @@
 // SPDX-License-Identifier: MIT
-// OpenZeppelin Contracts v4.4.1 (interfaces/draft-IERC2612.sol)
 
 pragma solidity ^0.8.0;
 
-import "../token/ERC20/extensions/draft-IERC20Permit.sol";
+// IERC2612 was marked as draft due to the EIP-2612 dependency.
```

Comment:
This message was added in ERC721Votes to explain that it was a draft due to EIP712 (712, not 721!), so it's not relevant in this case.

```suggestion
```